src/foundation.cs

```diff
@@ -12867,6 +12867,7 @@ interface NSTask {
 
 		// Fields
 		[Field ("NSTaskDidTerminateNotification")]
+		[Notification]
 		NSString NSTaskDidTerminateNotification { get; }
```

Comment:
same here, the name is incorrect. We must keep it (for compatibility) but the new API should not _suffer_ from it